### ISSUE TITLE
ConvertTo-Json : The maximum depth allowed for serialization is 100

### ISSUE THREAD

**Issue by iainbrighton** [devblackops/NetScaler#17]
Testing the module on a PowerShell v5.1 instance and it looks like the maximum `-Depth` value has changed to 100.

## Current Behavior
```powershell
PS C:\> ipmo netscaler -Force
PS C:\> $ns = Connect-NetScaler -IPAddress 10.200.0.100 -Credential nsroot -PassThru
PS C:\> Set-NSHostname -Session $ns -Hostname ns1.lab.local -Verbose -Force
VERBOSE: URI: http://10.200.0.100/nitro/v1/config/nshostname
ConvertTo-Json : The maximum depth allowed for serialization is 100.
At C:\Program Files\WindowsPowerShell\Modules\netscaler\Private\_InvokeNSRestApi.ps1:123 char:24
+ ... onPayload = ConvertTo-Json -InputObject $hashtablePayload -Depth ([in ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [ConvertTo-Json], InvalidOperationException
    + FullyQualifiedErrorId : ReachedMaximumDepthAllowed,Microsoft.PowerShell.Commands.ConvertToJsonCommand

VERBOSE: JSON Payload:
Invoke-RestMethod : {"errorcode":-1,"message":"Invalid payload format. Default format is json","severity":"ERROR"}
At C:\Program Files\WindowsPowerShell\Modules\netscaler\Private\_InvokeNSRestApi.ps1:144 char:21
+         $response = Invoke-RestMethod @restParams
+                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-RestMethod], WebExc
   eption
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand
```

## Your Environment
* Module version used: 1.3.0
* Operating System and PowerShell version: Server 2016 TP5 with 5.1.14300.1000

**Comment by iainbrighton** [comment]
__I'm happy to submit a PR for this if you want me to?__

**Comment by liamkennedy89** [comment]
Hey. I'm going through the 'Scripting: Automating NetScaler configurations using NITRO REST API and Powershell' (http://tinyurl.com/jcsc7fa) and I'm experiencing this issue when trying the following line:

`Restart-NSAppliance -NSSession $myNSSession -SaveNSConfig -WarmReboot -Wait`

Netscaler v 11.0
PS v 5.1

Any ideas?

**Comment by devblackops** [comment]
Can you file an issue? Also what version of the module are you running?
This should have been fixed in v1.3.0.

**Comment by liamkennedy89** [comment]
Sorry, mistakenly posted this in the wrong repo

________________________________
